### PR TITLE
fix: leaf clipping + scrollbar

### PR DESCRIPTION
**Pull request by jackyzha0**

### CODE REVIEW

File: src/pages/Home.js [hunk 1]
```diff
@@ -10,6 +10,8 @@ import footer_decal from '../assets/cmdf_footerdecal.svg'
 const Decal = styled.img`
   pointer-events: none;
   object-fit: cover;
+  overflow: hidden;
+  z-index: -1;
 `
 
 const TopDecal = styled(Decal)`
```
Comment (jvssicawu) [inline src/pages/Home.js:17]: can we change the 275px in left: and width: on lines 19 and 21 to 265px so that one lilypad isn't floating in space hehe

**Review by jvssicawu — CHANGES_REQUESTED** [review]

**Comment by jvssicawu** [comment]
LOL jacky u r gonna hate this but the nested scrollbar came back, I think u have to hide the overflow in the parent div of the decal :sweats:

**Comment by jackyzha0** [comment]
> LOL jacky u r gonna hate this but the nested scrollbar came back, I think u have to hide the overflow in the parent div of the decal :sweats:

the parent div of the decal is the whole page :mildpanic:

**Review by ianmah — COMMENTED** [review]
Quicklinks arent clickable for me on the homepage. (1080p monitor)

**Comment by ianmah** [comment]
image
maybe we can add a background colour to these so the dotted lines dont show?

**Comment by ianmah** [comment]
Not sure if because of this PR but i dont see a spinner when i click show spinner on charcuterie

**Review by jvssicawu — APPROVED** [review]
lgtm :))

**Pull request merged**